app/views/common_parameters/_parent_parameter.html.erb

```diff
@@ -11,7 +11,7 @@
       <% next if @hostgroup.group_parameters.map(&:name).include? parameter.name %>
       <tr>
         <td>
-          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :value => parameter.name, :readonly => true) %>
```

Comment:
This text field is used in the controller when overriding these parameters, so now parent parameter overrides don't work.  #2649 refactors this and will fix it anyway.